tools/EventClients/Clients/KodiSend/kodi-send.py

```diff
@@ -45,13 +48,17 @@ def usage():
     print("\t--port=PORT\t\t\tChoose what PORT to connect to (default=9777)")
     print("\t--keymap=KEYMAP\t\t\tChoose which KEYMAP to use for key presses (default=KB)")
     print('\t--button=BUTTON\t\t\tSends a key press event to Kodi, this option can be added multiple times to create a macro')
+    print('\t--mouse=X,Y\t\t\tSends the mouse position to Kodi')
+    print("\t--log=MESSAGE\t\t\tSends a log message to Kodi")
+    print("\t--loglevel=LEVEL\t\tSets the log level when using --log= (default=0)")
+    print("\t--notification=MESSAGE\t\tSends a notification to Kodi")
```

Comment:
While at it minding changing the line below to use Kodi instead of XBMC on the description?